packages/components/autocomplete/src/use-autocomplete.tsx

```diff
@@ -1422,6 +1419,10 @@ export type UseAutocompleteOptionGroupProps = HTMLUIProps<"ul"> & {
    * The label of the autocomplete option group.
    */
   label: string
+  /**
+   * Props for autocomplete option group element.
+   */
+  labelProps?: HTMLUIProps<"span">
```

Comment:
Do not define props to customize components here.  Set to the `autocomplete-option-group.tsx` side.

Reply:
fixed